testing/test_transfer_learning.py

```diff
@@ -10,9 +10,10 @@
 OUT_CHANNEL = 1
 INITIAL_LR = 0.001
 
-@pytest.mark.parametrize('fraction', [0.1, 0.3, 0.5])
+
+@pytest.mark.parametrize('fraction', [0.1, 0.2, 0.3])
```

Comment:
That's because we are using a very tiny network.. is it correct?

Reply:
Yes that is the reason.